examples/codon.ipynb

```diff
@@ -0,0 +1,213 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SynBioPython codon usage module"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates the functionalities of the codon usage module of [SynBiopython](https://synbiopython.org)."
```

Comment:
Note that the https link (https://synbiopython.org) is not redirected, only the standard http. See also in setup.py

Reply:
Thanks, @vegh. I've updated the redirect such that both http and https redirect successfully.